README.md

```diff
@@ -31,16 +31,18 @@ Prerequisites
 ---------
  * [Java 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) (Java 9+ are not supported, will get compiler issue)
  * [Git](https://git-scm.com/downloads)
+ * [Gradle](https://gradle.org/install/)
```

Comment:
User do not have to install Gradle. Because there is  gradle wrapper inside.

Reply:
oh okay